kano_settings/system/advanced.py

```diff
@@ -368,5 +368,14 @@ def set_parental_level(level_setting):
         set_ultimate_parental(False)
 
     blacklist, whitelist = read_listed_sites()
+
+    # SafeSearch: disabling google.com, until we have higher Chromium version
+    # that implements policy files.
+    if 'chromium' in enabled:
+        if blacklist:
```

Comment:
check this line for the 'www' thing: https://github.com/KanoComputing/kano-settings/blob/master/kano_settings/system/advanced.py#L142-L149